pandaserver/srvcore/allowed_methods.py

```diff
@@ -20,10 +20,6 @@
     "put_file_recovery_request",
     "put_workflow_request",
 ]
-
-# methods from pandaserver.dataservice.DataService
-allowed_methods += ["datasetCompleted", "updateFileStatusInDisp"]
```

Comment:
watch for errors in panda-Entry.log once you change this. I think it would trigger "403 Forbidden"

Reply:
`[ekaravak@aiadm09 ~]$ wassh --host 'atlas-pandaserver-0[0-8]' -l root 'grep -ri Forbidden /var/log/panda/panda-Entry.log'
aipanda090.cern.ch:
aipanda092.cern.ch:
aipanda093.cern.ch:
aipanda094.cern.ch:
aipanda095.cern.ch:
aipanda096.cern.ch:
aipanda097.cern.ch:
aipanda098.cern.ch:
aipanda099.cern.ch:
`
also checked for datasetCompleted and updateFileStatisInDisp, no results.